index.js

```diff
@@ -80,6 +80,19 @@ app.post('/start', (req, res) => {
   if (req.body.passphrase) {
     walletConfig['passphrase'] = req.body.passphrase;
   }
+  const walletID = req.body['wallet-id'];
+
+  if (walletID in wallets) {
+    // We already have a wallet for this key
+    // so we log that it won't start a new one because
+    // it must first stop the old wallet and then start the new
+    console.log('Error starting wallet because this wallet-id is already used. You must stop the wallet using it first.');
```

Comment:
Typo: `... wallet-id is already in use. You must stop the wallet first.`

Reply:
Fixed, thanks